build-tools/tasks/styles.js

```diff
@@ -78,6 +78,7 @@ function stylesTask(theme) {
       designTokensFileName: theme.designTokensOutput,
       descriptions,
       jsonSchema: true,
+      failOnDeprecations: true,
```

Comment:
Currently there are zero deprecation warnings. This line is to make sure it the build fails if anybody attempts using deprecated setup